### PR TITLE
For commenting on 01-00-Understanding-an-astronomical-CCD-image.ipynb (part of review-ee0cbc6)

### PR DESCRIPTION
**Pull request by mwcraig**

### CODE REVIEW

File: notebooks/01-00-Understanding-an-astronomical-CCD-image.ipynb [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    "But the ideal scenario does not in fact hold true. A solid understanding of\n",
+    "*why* pixel values are not directly proportional to light is useful before\n",
+    "diving into the details of image reduction."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Counts, photons, and electrons\n",
```
Comment (isobelsnellenberger) [inline notebooks/01-00-Understanding-an-astronomical-CCD-image.ipynb]: test comment

Comment (kakirastern) [reply notebooks/01-00-Understanding-an-astronomical-CCD-image.ipynb]: I think the guide is very good in terms of coverage and presentation, especially in the hands-on examples present. However, I really would want to see a bit more of the background information, such as to provide the **rationale** or the **motivation** for the performance of the tasks concerned.

File: notebooks/01-00-Understanding-an-astronomical-CCD-image.ipynb [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+   "metadata": {},
+   "source": [
+    "# Understanding an astronomical CCD image\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An astronomical image like the one shown below is essentually a two-dimensional\n",
```
Comment (richardhendricks) [inline notebooks/01-00-Understanding-an-astronomical-CCD-image.ipynb]: essentually -> essentially

Comment (richardhendricks) [reply notebooks/01-00-Understanding-an-astronomical-CCD-image.ipynb]: Should this cover other image artifacts like saturation trails and diffraction spikes?

Comment (mwcraig) [reply notebooks/01-00-Understanding-an-astronomical-CCD-image.ipynb]: Yes, it should, thanks for the suggestion!

Comment (mwcraig) [reply notebooks/01-00-Understanding-an-astronomical-CCD-image.ipynb]: FYI, I've added an issue as a reminder to add this: https://github.com/mwcraig/ccd-reduction-and-photometry-guide/issues/138

I'm going to close this commenting pull request in preparation for publishing a revised version (it also includes that typo fix you caught).

**Pull request closed without merging**